open_dread_rando/files/lua_libraries/room_names.lua

```diff
@@ -0,0 +1,527 @@
+RoomNameGui = RoomNameGui or {
+    cameraDict = nil
+}
+
+function RoomNameGui.BuildCameraDict()
+	if RoomNameGui.cameraDict ~= nil then
+		Game.LogWarn(0, "Camera dict exists!")
+		return
+	end
+
+	local artaria = {
+		collision_camera_000 = "First Tutorial",
```

Comment:
I wonder if this should be provided from Randovania instead, to make sure it's aligned. Not sure if worth the effort though.

Reply:
ah, though it *is* worth noting that a couple of rooms would need special handling that way, since they're a single collision camera split into multiple rooms in the db. might not be as simple then

Reply:
I agree that it should be done but it would require some more substantial changes that may take a bit, as i'm not sure how to import a python dict into a lua table. as well as the special cases (I think it's Burenia - Main Hub Tower Top/Middle/Bottom and Ghavoran - Golzuna Tower/Teleport to Cataris). 

I'm thinking I'll split the lua table into a separate file, add this, then work on that in a separate PR.